scripts/scil_mrds_metrics.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to compute FA/MD/RD/AD for each tensor solution of MRDS.
+It will output the results in 4 different 4D files.
+"""
+
+import logging
+import numpy as np
+import nibabel as nib
+import argparse
+
+from dipy.reconst.dti import fractional_anisotropy
+
+from scilpy.io.image import get_data_as_mask
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_verbose_arg,
+                             assert_inputs_exist, assert_outputs_exist,
+                             assert_headers_compatible)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_eigenvalues',
+                   help='MRDS eigenvalues file.')
+
+    p.add_argument('--mask',
+                   help='Path to a binary mask.\nOnly data inside '
+                        'the mask will be used for computations and '
+                        'reconstruction. (Default: %(default)s)')
+
+    p.add_argument(
+        '--not_all', action='store_true', dest='not_all',
```

Comment:
The dest is not useful, same name as the initial variable.

Idem for all options below.